### DISCUSSION TITLE
UERANSIM Development Fund Raising

### DISCUSSION THREAD

**Discussion by infinitydon** [aligungr/UERANSIM#460] [General]
Hi everyone,

Please I want to propose that we can do some sort of fund raising to accelerate the development of ueransim with more features, the fund can go towards main area of getting some dedicated developers that can with @aligungr .

I believe several people are benefitting from this project and I think this will go a long way in encouraging Telco Opensource software like UERANSIM.

If anyone has any ideas/thoughts about this, kindly share.

**Comment by infinitydon** [comment]
@aligungr - What is your opinion if I try to do a fund raising for UERANSIM? How much do you think is needed to add new features/improvement to the project? Also if you can list the most important features/improvements that you think should be incorporated into UERANSIM, kindly provide it..

I intend on using gofundMe website for this..

Or can Open Collective be used for the same purpose?

**Reply by aligungr** [reply]
Thanks for the consideration @infinitydon.

I think the most important features might be
- Complete radio interface
- QoS flow handling and SDAP implementation
- Performance optimizations for data plane

I believe that after this point the radio interface will remain simulated. Because when I looked the general use case of the UERANSIM majority of the users only care for the N1/N2/N3 interfaces.

I think the only fundamental missing features are QoS flow handling and implementation of SDAP protocol. Also there is a need for performance optimizations for data plane for sure.

See https://github.com/aligungr/UERANSIM/issues/353#issuecomment-866970813 for other features *might* be implemented soon or later.

> Or can Open Collective be used for the same purpose?

Personally it does not matter if it's GoFundMe or OpenCollective
